app/validators/school_certifying_official_validator.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class SchoolCertifyingOfficialValidator < ActiveModel::Validator
+  REQUIREMENT_DESCRIPTIONS = ["Valid priority values: #{SchoolCertifyingOfficial::VALID_PRIORITY_VALUES}"].freeze
```

Comment:
Is REQUIREMENT_DESCRIPTIONS used anywhere?

Reply:
https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/app/controllers/uploads_controller.rb#L47

Reply:
Ah, I see it's used in the uploads controller

Reply:
Yes, it's used on the upload page for the related model:

![image](https://user-images.githubusercontent.com/41960449/82212838-31df4200-98e1-11ea-827f-d47842e3e44c.png)